__tests__/lib/search-and-replace.js

```diff
@@ -26,6 +27,8 @@ const testFilePath = path.resolve( fixtureDir, 'client-file-uploader', 'tinyfile
 jest.mock( 'node-fetch' );
 fetch.mockReturnValue( Promise.resolve( new Response( 'ok' ) ) );
 
+jest.spyOn( apiConfig, 'checkIfUserIsVip' ).mockResolvedValue( true );
```

Comment:
This feels annoying if we have to remember to do it for every test suite (it will lead to inevitably lead to wasted time wondering "why is my test suite failing").

Can we globally mock the module instead?

Reply:
@mjangda : This is done in 0deca2f

had to create a separate setup file for jest mocks because it was messing up the env variable mocks in the `jest.setup.js` file

Reply:
thanks! good point!